src/beacon-api/app/schemas/diagnostics.py

```
@@ -0,0 +1,179 @@
+    name: str
+    severity: str = "MEDIUM"
+    evaluation_logic: Optional[Dict[str, Any]] = None
+    description: Optional[str] = None
+
+    @field_validator("evaluation_logic", mode="before")
+    @classmethod
+    def parse_eval_logic(cls, v: Any) -> Any:
+        if isinstance(v, str):
+            try:
+                return json.loads(v)
+            except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)